openpgp/packet/notation.go

```diff
@@ -0,0 +1,29 @@
+package packet
+
+// Notation type represents a Notation Data subpacket
+// see https://tools.ietf.org/html/rfc4880#section-5.2.3.16
+type Notation struct {
+	Name string
+	Value []byte
+	IsCritical bool
+	IsHumanReadable bool
+}
+
+func (not *Notation) getData() []byte {
```

Comment:
I'd find it easier to read if the variable was not called 'not'

Reply:
:yum: Hmhm :+1: Renamed it to `notation` .